gammapy/data/event_list.py

```diff
@@ -132,7 +132,14 @@ def altaz(self):
 
         lon, lat = self['AZ'], self['ALT']
         return SkyCoord(lon, lat, unit='deg', frame=altaz_frame)
-
+    
+    @property
+    def offset(self):
```

Comment:
ok I am sorry I am very sick today so I will leave and continue this changes tomorrow…
I replace 
ev_detx, ev_dety = self['DETX'], self['DETY']
offset= np.sqrt(ev_detx ** 2 + ev_dety ** 2)	

by
position = self.radec
center=self.target_radec
offset=center.separation(position)

but that doesn’t give the same result….
Le 21 janv. 2016 à 17:15, Christoph Deil <notifications@github.com> a écrit :

> In gammapy/data/event_list.py:
> 
> > @@ -132,7 +132,14 @@ def altaz(self):
> >  
> >          lon, lat = self['AZ'], self['ALT']
> >          return SkyCoord(lon, lat, unit='deg', frame=altaz_frame)
> > -
> > +    
> > +    @property
> > +    def offset(self):
> ok so position will be self.radec and the center will be pointing_radec?
> 
> Yes.
> 
> —
> Reply to this email directly or view it on GitHub.
>